main.js

```diff
@@ -0,0 +1,270 @@
+      dom.p1Section.classList.add('active-player');
+      dom.p2Section.classList.remove('active-player');
+    },
+
+    getDOMstrings() {
+      return dom;
+    },
+
+    clearForm() {
+      dom.form.reset();
+    },
+
+    hideForm() {
+      dom.form.classList.toggle('invisible');
+    },
+  };
+})();
+
+const gameLogic = ((UICtrl) => {
+  const DOM = UICtrl.getDOMstrings();
+  let gameBoardArr = ['', '', '', '', '', '', '', '', ''];
+  let player1; let player2; let current; let passivePlayer;
+  let counter = 0;
+
+  const createPlayers = (playersInfo) => {
+    player1 = player(playersInfo.player1Name, playersInfo.player1Sym, true);
+    player2 = player(playersInfo.player2Name, playersInfo.player2Sym);
+  };
+
+  const setPlayers = () => {
+    const inputs = UICtrl.getPlayers();
+    if (UICtrl.validatePlayerSymbol() === true) {
+      createPlayers(inputs);
+      UICtrl.displayPlayerInfo(inputs);
+      UICtrl.displayScore(player1.score, player2.score);
+      UICtrl.clearForm();
+      UICtrl.hideForm();
+    }
+  };
+
+  const currentPlayer = () => {
+    current = player1.isTurn === true ? player1 : player2;
+    passivePlayer = current === player2 ? player1 : player2;
+  };
+
+  const togglePlayer = () => {
+    counter += 1;
+    if (current === player1) {
+      player1.isTurn = false;
+      player2.isTurn = true;
+    } else {
+      player2.isTurn = false;
+      player1.isTurn = true;
+    }
+  };
+
+  const checkWinner = () => {
+    const winingComposition = [['1', '2', '3'], ['1', '4', '7'], ['1', '5', '9'], ['2', '5', '8'], ['3', '6', '9'], ['4', '5', '6'], ['7', '8', '9'], ['3', '5', '7']];
+    // eslint-disable-next-line no-restricted-syntax
+    for (const item of winingComposition) {
+      let count = 0;
+      for (let i = 0; i < 3; i += 1) {
+        if (current.playerChoices.includes(item[i])) {
+          count += 1;
+        }
+      }
+      if (count === 3) {
+        return true;
+      }
+    } return false;
+  };
+
+  const checkDraw = () => {
+    if (!gameBoardArr.includes('') && checkWinner() !== true) {
+      return true;
+    }
+    return false;
+  };
+
+  const resetBoard = () => {
+    gameBoardArr = ['', '', '', '', '', '', '', '', ''];
+  };
+
+  const resetValues = () => {
+    counter = 0;
+    player1.isTurn = true;
+    player2.isTurn = false;
+    player1.playerChoices = [];
+    player2.playerChoices = [];
+    UICtrl.resetPlayers();
+  };
+
+  const newRound = () => {
+    resetBoard();
+    resetValues();
+    UICtrl.displaySym(gameBoardArr);
+    UICtrl.displayScore(player1.score, player2.score);
+  };
+
+  const result = () => {
+    if (counter >= 5 && checkWinner() === true) {
+      UICtrl.displayResult(current);
+      current.score += 1;
+      UICtrl.displayScore(player1.score, player2.score);
+      newRound();
+    } else if (counter === 9 && checkDraw() === true) {
+      UICtrl.displayResult(true);
+    }
+  };
+
+  const resetGame = () => {
+    location.reload();
+  };
```

Comment:
add this linter config comment (do this): 
```
  /* eslint no-restricted-globals: ["error", "event"] */
  const resetGame = () => {
    location.reload();
  };
```
documentation about this linter error here --> [click this](https://eslint.org/docs/rules/no-restricted-globals)